src/_drawColorScale.js

```diff
@@ -10,7 +10,7 @@ export default function() {
 
   const data = this._data;
 
-  const position = this._colorScalePosition.bind(this)(this.config()) || "bottom";
+  const position = this._colorScalePosition.bind(this)(this.config());
```

Comment:
I read your comments in the issue, and I understand your points. The `|| "bottom"` catch was originally there as a fallback in case the user had not set a colorScalePosition value, but since I changed that value to be a _Function_, this fallback needs to change.

Basically, if `position` is either `undefined` or not a string we recognize, we should set position to "bottom". Can you add something like this and see if it works as intended?

```js
let position = this._colorScalePosition.bind(this)(this.config());
if (![false, "top", "bottom", "left", "right"].includes(position)) position = "bottom";
```

This logic may also need to be ported over to `_drawLegend.js`, so can you check there as well?
```

Reply:
I'm checking this solution and it works if position is `right, left, top, bottom` or `false` but not if it is `undefined` or not a string we recognize. In those cases we get a visualization without color scale as in the image (`colorScalePosition: undefined`).
![image](https://user-images.githubusercontent.com/39899484/112366007-0ffbdc00-8cb7-11eb-8690-ae3f433b42f7.png)
Looking beyond i found that in [Viz.js](https://github.com/d3plus/d3plus-viz/blob/master/src/Viz.js#L428) we also set `legendPosition` and `colorScalePosition` which i think are related with this bug.
And adding the same logic to those lines seems to work when we set `undefined` or even `asdf` on colorScalePosition (as second image) and also expected values (`right, left, top, bottom and false`) as in third gif (starting from `undefined` to `false`).

```
let legendPosition = this._legendPosition.bind(this)(this.config());
if (![false, "top", "bottom", "left", "right"].includes(legendPosition)) legendPosition = "bottom";
let colorScalePosition = this._colorScalePosition.bind(this)(this.config());
if (![false, "top", "bottom", "left", "right"].includes(colorScalePosition)) colorScalePosition = "bottom";
```
![image](https://user-images.githubusercontent.com/39899484/112367576-de841000-8cb8-11eb-97bc-07fedfedfefc.png)

![colorscale-2](https://user-images.githubusercontent.com/39899484/112368610-07f16b80-8cba-11eb-87c4-0369a77c659f.gif)